tests/phpunit/SnowflakeTest.php

```diff
@@ -983,32 +984,6 @@ public function simpleTableColumnsDataProvider(): array
                     $dbSchema
                 ),
             ],
-            'test simplePDO query id column and previos state and no limit' => [
-                [
-                    'table' => [
-                        'tableName' => 'auto_increment_timestamp',
-                        'schema' => $dbSchema,
-                    ],
-                    'columns' => [
-                        'id',
-                        'name',
-                        'number',
-                        'timestamp',
-                    ],
-                    'incrementalFetchingLimit' => 0,
```

Comment:
je to otestovane niekde inde? Imho by tam nejaky test bolo dobre nechat, ... bude to niekto prerabat a nebude o tom vediet.

Reply:
jj to musí být lae přes datadir testy.... takže v dalším PR

Reply:
ok, v pohode